tf_agents/environments/gym_wrapper_test.py

```diff
@@ -436,6 +436,13 @@ def test_obs_dtype(self):
     time_step = env.reset()
     self.assertEqual(env.observation_spec().dtype, time_step.observation.dtype)
 
+  def test_reward_range(self):
+    cartpole_env = gym.spec('CartPole-v1').make()
+    cartpole_env.reward_range = (-3.0, 5.0)
```

Comment:
Are there any gym environment that already has a reward_range?

Reply:
Checking the OpenAI depot, only Frozen Lake (https://github.com/openai/gym/search?q=reward_range)
It's easier for Muzero, as you can declare game environment range but after test on many environment I automatically update reward range with the maximum discovered